app/routes.js

```diff
@@ -73,6 +74,12 @@ export const routes = (authProvider) => {
     res.redirect(`/solutions/capabilities-selector${capabilitiesParam}`);
   }));
 
+  router.get('/solutions/covid19', withCatch(async (req, res) => {
```

Comment:
add a TODO with the name of the featureFlag like we have done elsewhere to quickly find the code to be deleted